pkg/notes/adaptor/repository/dummy.ts

```diff
@@ -124,3 +126,40 @@ export class InMemoryBookmarkRepository implements BookmarkRepository {
     return Promise.resolve(Option.some(bookmarks));
   }
 }
+
+export class InMemoryNoteAttachmentRepository
+  implements NoteAttachmentRepository
+{
+  private readonly attachments: Map<NoteID, MediumID[]>;
+  private readonly medium: Map<MediumID, Medium>;
+
+  constructor(medium: Medium[]) {
+    this.attachments = new Map();
+    this.medium = new Map(medium.map((m) => [m.getId(), m]));
+  }
+
+  async create(
+    noteID: NoteID,
+    attachmentFileID: MediumID[],
+  ): Promise<Result.Result<Error, void>> {
+    if (attachmentFileID.map((v) => this.medium.has(v)).includes(false)) {
```

Comment:
```suggestion
    if (!attachmentFileID.every((v) => this.medium.has(v))) {
```
`includes(false)` よりも `every` などで判定したほうがよさそうです.